core/src/main/scala/better/files/ReaderInputStream.scala

```diff
@@ -51,7 +51,8 @@ class ReaderInputStream(reader: Reader, encoder: CharsetEncoder, bufferSize: Int
     encoderOut.flip()
   }
 
-  override def read(b: Array[Byte], off: Int, len: Int) = {
+  override def read(_b: Array[Byte] | Null, off: Int, len: Int) = {
+    val b = _b.nn
```

Comment:
**Overridden java arguments are never null**
Argument should never be null, immediately remove nullness of parameter.